front_end/messages/es.json

```
@@ -1188,6 +1188,27 @@
+  "exportCharts": "Export Charts",
+  "downloadExport": "Download",
+  "downloadAll": "Download All",
+  "availableCharts": "Available charts",
+  "radialForecastDisplay": "Radial Forecast Display",
+  "forecastTimeline": "Forecast Timeline",
+  "histogram": "Histogram",
+  "fullSizePdf": "Full-size PDF",
+  "fullSizeCdf": "Full-size CDF",
+  "tableWithTop4": "Table with top 4 + others",
+  "fullTable": "Full table",
+  "timeline": "Timeline",
```

Biome (2.4.11)

[error] The key timeline was already declared.

(lint/suspicious/noDuplicateObjectKeys)